pyanaconda/packaging/dnfpayload.py

```diff
@@ -115,8 +115,8 @@ def reasonable_mpoint(mpoint):
                    # for root we need to take in count both download and install size
                    if ((key != root_mpoint and val > requested)
                    or val > requested_root) and reasonable_mpoint(key)}
-    log.debug('Estimated size: download %s & install %s - df: %s', requested,
-              (requested_root - requested), df)
+    log.debug('Estimated size: download %s & install %s', requested,
+              (requested_root - requested))
```

Comment:
This will remove 3 or more lines in the log of filesystem mountpoint which won't be used for downloading. 
I want to make logs more readable so I removed this. If we want these logs we want better formating or something though. I can try to create formatted version of this.